pkgs/by-name/li/libpff/package.nix

```
@@ -8,11 +8,11 @@
 
 stdenv.mkDerivation rec {
   pname = "libpff";
-  version = "20211114";
+  version = "20231205";
 
   src = fetchzip {
     url = "https://github.com/libyal/libpff/releases/download/${version}/libpff-alpha-${version}.tar.gz";
-    sha256 = "sha256-UmGRBgi78nDSuuOXi/WmODojWU5AbQGKNQwLseoh714=";
+    sha256 = "sha256-VrdfZRC2iwTfv3YrObQvIH9QZPTi9pUQoAyUcBVJyes=";
```

Optional but nice to have:
```suggestion
    hash = "sha256-VrdfZRC2iwTfv3YrObQvIH9QZPTi9pUQoAyUcBVJyes=";
```